assets/openobserve/Dockerfile

```
@@ -0,0 +1,24 @@
+# This file is used for building the container, ensuring installation of the required tool and
+# dependencies
+
+# If there is any dedicated image available, you should build the benchmarking image on top of that
+FROM public.ecr.aws/zinclabs/openobserve:v0.15.0-rc1
+
+FROM ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-jammy:main
+
+COPY --from=0 /openobserve /openobserve
```

Hadolint (2.12.0)

[warning] `COPY --from` should reference a previously defined `FROM` alias

(DL3022)